climate_categories/data/PRIMAP_entity.yaml

```diff
@@ -0,0 +1,1627 @@
+    comment: Emissions rate of the refrigerant HCFC503, which is a mixture of 40.1%
+      HFC23, 59.9% CFC13.
+    info:
+      addition_rule: extensive
+  HCFC504:
+    title: HCFC504
+    comment: Emissions rate of the refrigerant HCFC504, which is a mixture of 48.2%
+      HFC32, 51.8% CFC115.
+    info:
+      addition_rule: extensive
+  HCFC505:
+    title: HCFC505
+    comment: Emissions rate of the refrigerant HCFC505, which is a mixture of 78%
+      CFC12, 22% HCFC31.
+    info:
+      addition_rule: extensive
+  HCFC506:
+    title: HCFC506
+    comment: Emissions rate of the refrigerant HCFC506, which is a mixture of 55.1%
+      HCFC31, 44.9% CFC114.
+    info:
+      addition_rule: extensive
+  HFC507a:
+    title: HFC507a
+    comment: Emissions rate of the refrigerant HFC507a, which is a mixture of 50%
+      HFC125, 50% HFC143a.
+    info:
+      addition_rule: extensive
+  HFC508a:
+    title: HFC508a
+    comment: Emissions rate of the refrigerant HFC508a, which is a mixture of 39%
+      HFC23, 61% C2F6.
+    info:
+      addition_rule: extensive
+  HFC508b:
+    title: HFC508b
+    comment: Emissions rate of the refrigerant HFC508b, which is a mixture of 46%
+      HFC23, 54% C2F6.
+    info:
+      addition_rule: extensive
+  HCFC509a:
+    title: HCFC509a
+    comment: Emissions rate of the refrigerant HCFC509a, which is a mixture of 44%
+      HCFC22, 56% C3F8.
+    info:
+      addition_rule: extensive
+  HC510a:
+    title: HC510a
+    comment: Emissions rate of the refrigerant HC510a, which is a mixture of 88% HCE170,
+      12% HC600a.
+    info:
+      addition_rule: extensive
+  HC511a:
+    title: HC511a
+    comment: Emissions rate of the refrigerant HC511a, which is a mixture of 95% C3H8,
+      5% HCE170.
+    info:
+      addition_rule: extensive
+  HFC512a:
+    title: HFC512a
+    comment: Emissions rate of the refrigerant HFC512a, which is a mixture of 5% HFC134a,
+      95% HFC152a.
+    info:
+      addition_rule: extensive
+  HFO513a:
+    title: HFO513a
+    comment: Emissions rate of the refrigerant HFO513a, which is a mixture of 56%
+      HFO1234yf, 44% HFC134a.
+    info:
+      addition_rule: extensive
+  HFO513b:
+    title: HFO513b
+    comment: Emissions rate of the refrigerant HFO513b, which is a mixture of 58.5%
+      HFO1234yf, 41.5% HFC134a.
+    info:
+      addition_rule: extensive
+  HCO514a:
+    title: HCO514a
+    comment: Emissions rate of the refrigerant HCO514a, which is a mixture of 74.7%
+      HFC1336mzz, 25.3% HCO1130.
+    info:
+      addition_rule: extensive
+  HFO515a:
+    title: HFO515a
+    comment: Emissions rate of the refrigerant HFO515a, which is a mixture of 88%
+      HFO1234ze, 12% HFC227ea.
+    info:
+      addition_rule: extensive
+  HFO516a:
+    title: HFO516a
+    comment: Emissions rate of the refrigerant HFO516a, which is a mixture of 77.5%
+      HFO1234yf, 8.5% HFC134a, 14% HFC152a.
+    info:
+      addition_rule: extensive
+  emissions:
```

Comment:
The gas baskets are still missing.

Reply:
Yes, gas baskets are missing. Do you have a good source for which gas baskets to include and what is part of the gas baskets?

Reply:
Regarding gas baskets having children and also be a child of emissions: Yes, that's possible. The hierarchy has `total_sum=False` because I expected that entities will not sum up nicely, in general. E.g. the mixtures and their constituents overlap somewhat.